src/confetti.js

```diff
@@ -311,14 +313,19 @@
   function updateFetti(context, fetti) {
     fetti.x += Math.cos(fetti.angle2D) * fetti.velocity + fetti.drift;
     fetti.y += Math.sin(fetti.angle2D) * fetti.velocity + fetti.gravity;
-    fetti.wobble += fetti.wobbleSpeed;
     fetti.velocity *= fetti.decay;
     fetti.tiltAngle += 0.1;
     fetti.tiltSin = Math.sin(fetti.tiltAngle);
     fetti.tiltCos = Math.cos(fetti.tiltAngle);
     fetti.random = Math.random() + 2;
-    fetti.wobbleX = fetti.x + ((10 * fetti.scalar) * Math.cos(fetti.wobble));
-    fetti.wobbleY = fetti.y + ((10 * fetti.scalar) * Math.sin(fetti.wobble));
+    fetti.wobbleX = fetti.x + (10 * fetti.scalar);
+    fetti.wobbleY = fetti.y + (10 * fetti.scalar);
```

Comment:
This math currently can get done twice. Since we are animating a lot of particles and trying to maintain a smooth effect, even a little bit of duplicate math can have a significant effect. If there is a conditional, we should put this default value in the `esle` to make sure we either do one version or the other but never both.

Reply:
You are absolutely right. Ive moved this inside the condition :)